litmus-portal/frontend/src/pages/Community/index.tsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import Paper from '@material-ui/core/Paper'; // Temporary -> Should be replaced with Chart
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import Button from '@material-ui/core/Button';
+import Grid from '@material-ui/core/Grid';
+import CustomInfoFilled from '../../components/CustomInfoFilled/index';
+import Scaffold from '../../containers/layouts/Scaffold/index';
+import useStyles from './styles';
+import QuickActionCard from '../../components/QuickActionCard';
+
+// Reusable Header Component
+const Header2: React.FC = ({ children }) => {
+  const classes = useStyles();
+  return (
+    <div>
+      <Typography className={classes.header2}>
+        <strong>{children}</strong>
+      </Typography>
+    </div>
+  );
+};
+
+const Community: React.FC = () => {
+  const classes = useStyles();
+  return (
+    <Scaffold>
+      <div>
+        <div>
+          <Typography className={classes.mainHeader}>Community</Typography>
+        </div>
+
+        {/* Litmus Daily Insights */}
+        <section className="Daily Insights">
+          <Header2>Daily Insights</Header2>
+          <Typography>
+            Stats for the Litmus community in the last 24 hours
+          </Typography>
+          <div className={classes.cardDiv}>
+            <CustomInfoFilled
+              color="#109B67"
+              value={12300}
+              plus
+              statType="Total chaos Operator Installed"
+            />
+            <CustomInfoFilled
+              color="#858CDD"
+              value={12300}
+              plus
+              statType="Total Experiments today"
+            />
+            <CustomInfoFilled
+              color="#F6B92B"
+              value={2500}
+              plus
+              statType="Total Runs experiments today"
+            />
+            <CustomInfoFilled
+              color="#BA3B34"
+              value={30000}
+              plus
+              statType="Total chaos experiments today"
+            />
+          </div>
+        </section>
+
+        {/* Litmus Analytics Dashboard */}
+        <section className="Litmus Analytics Dashboard">
+          <div className={classes.LitmusAnalyticsBlock}>
+            <Header2>Periodic growth of Litmus</Header2>
+            <Grid container spacing={3}>
+              <Grid item xs={12} sm={12} md={8}>
+                <Paper className={classes.paper}>Dummy Graph Analytics</Paper>
+              </Grid>
+              <Grid item xs={12} sm={12} md={4}>
```

Comment:
Grid uses css flexbox under the hood....maybe we can use flexbox in css to style this...this will let us avoid touching the markup if we need to make breakpoints and change styling...also lesser import means lesser bundle size.